### ISSUE TITLE
Delay in response using Sapi

### ISSUE THREAD

**Issue by Ambro86** [dkager/tolk#15]
Hi, first of all thanks for this project, which I used with Java. It works great with Jaws and Nvda, but I notice a greater slowness when using Sapi. Is there a way to reduce it?
Thanks.

**Comment by TheQuinbox** [comment]
No, that's just how SAPI works in general. You'll notice it if you use SAPI with NVDA too.
